examples/sushi/models/waiter_as_customer_by_day.sql

```diff
@@ -6,6 +6,7 @@ MODEL (
   ),
   owner jen,
   cron '@daily',
+  start '1 week ago',
```

Comment:
let's not add 1 week everywhere, i purposefully didn't add all of them to make it so we test inference of start

Reply:
Well adding plan start is not proper since prod environments cannot represent a subset of data

Reply:
how is that relevant? 

the start date of a model should inherit from its parents

Reply:
Oh I misunderstood what you said. So make sure all root models have it only.